### ISSUE TITLE
Refund request

### ISSUE THREAD

**Issue by kdpnetstake** [EvernodeXRPL/evernode-host#167]
Hello team, 
Can you please refund me the 250 EVR lost from this account rs57tm5iJ4PYEi2To4PSiYBj5uM2jq9pTg . This was due to a failed initial installation. Many thanks... Kdpnetstake

**Comment by chalith** [comment]
@kdpnetstake do you have the log file generated during the failure of installation? If you have can you attach it here for our reference?

**Comment by kdpnetstake** [comment]
Hi, the content are as follows:


---Sashimono installer (upgrade)---
STAGE Installing docker packages
Installing rootless docker packages into /usr/bin/sashimono/dockerbin
Private docker registry installation skipped
Configuaring host setup on Evernode...
useradd: warning: the home directory /home/sashimbxrpl already exists.
useradd: Not copying any file from skel directory into it.
STAGE Configuring Sashimono services
Creating cgroup rules...
Configuring sashimono agent service...
Existing Sashimono data directory found. Updating...
Created symlink /etc/systemd/system/multi-user.target.wants/sashimono-cgcreate.service → /etc/systemd/system/sashimono-cgcreate.service.
Created symlink /etc/systemd/system/multi-user.target.wants/sashimono-agent.service → /etc/systemd/system/sashimono-agent.service.
Installing Evernode Xahau message board...
Updated mb user .bashrc.
STAGE Configuring Xahau message board service
Created symlink /home/sashimbxrpl/.config/systemd/user/default.target.wants/sashimono-mb-xrpl.service → /home/sashimbxrpl/.config/systemd/user/sashimono-mb-xrpl.service.
Installed Evernode Xahau message board.
Starting the sashimono and message board services.
Sashimono installed successfully.

Basically, I accidently did a copypaste command on a different screen which cut the installation short. I went back in and just started from where I left off. The installation went ahead fine but then I found I had lost 250 EVR.  Having purchased more EVR in order to move ahead I then had a successful registration.  The installation log shows only the successful registration and not the failed attempts. 
Thanks, kdpnetstake

**Comment by kdpnetstake** [comment]
Hello, if you need any more info from me please let me know. Otherwise, can you refund the 250 EVR.  
Thanks

**Comment by chalith** [comment]
Going through the pending refunds. We'll do the refund today

**Comment by chalith** [comment]
We are going through the refunds again. Will do the refund today

**Comment by chalith** [comment]
Hope you receive your refund.
Hashes:
4FE30D1C880DC812F7F7D064126568BCB8134D9D029A210897F8829145577D35
D5ED143163C34BBB750E3354172270E41CE7711C92E1BEDFBFBCE7C7F9644C1D

**Comment by kdpnetstake** [comment]
Thanks very much. Received nicely....
Have a great day!
